### PR TITLE
Uswds-navbar-v2

### PR DESCRIPTION
**Pull request by ojbravo**
## Changes

Added USWDS Navbar w/ Algolia Search

## Testing

Increased added 5s timeout to Filing.spec test to ensure it passes.

### CODE REVIEW

**Review by meissadia — CHANGES_REQUESTED** [review]
- User name + logout wrap awkwardly on narrow screens

  Screen Shot 2023-05-22 at 5 03 01 PM

File: cypress/e2e/filing/Filing.spec.js [hunk 1]
```diff
@@ -145,13 +145,14 @@ describe("Filing", function() {
 
             // Action: Review Quality Edits
             cy.get('.NavButtonContainer > .NavButton').click()
-            cy.wait(ACTION_DELAY)
+            cy.wait(5000)
```
[the remark this answers is not in the document]
Comment (ojbravo) [reply cypress/e2e/filing/Filing.spec.js:148]: Update made.

**Comment by ojbravo** [comment]
Corrected logout location via css update, added NAV_DELAY test variable, and removed commented code. Will work on simplifying <ShowUserName /> component.

File: src/common/ShowUserName.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  
+  return (
+    <>
+      {isLoggedIn ? (
+        <div className='user'>{userName} <button onClick={handleLogout}>Logout</button></div>
+      ) : null}
+    </>
+  )
+}
```
[the remark this answers is not in the document]
Comment (ojbravo) [reply src/common/ShowUserName.jsx:28]: I'll look into this one... Good looking out!

File: src/common/Header.jsx [hunk 3]
```diff
@@ -15,49 +25,99 @@ export const hideHeaderFooter = (path) => {
...
+                  <li key={link.name} className="usa-nav__primary-item">
+                    {!link.submenu ? 
+                      <Link
+                        to={link.href}
+                        className={
+                            isActive
+                            ? 'usa-nav__link usa-current'
+                            : 'usa-nav__link'
+                        }
+                        //target={link.name === 'Filing' ? '_blank' : null}
```
Comment (meissadia) [inline src/common/Header.jsx:74]:
```suggestion
```

**Review by Michaeldremy — APPROVED** [review]
Look good to me!

File: src/common/Header.jsx [hunk 4]
```diff
@@ -15,49 +25,99 @@ export const hideHeaderFooter = (path) => {
...
+                                  to={sublink.href}
+                                >
+                                  {sublink.name}
+                                </Link>
+                            }
+                          </li>)
+                          )}
+                        </ul>
+                      </>
+                    }
```
Comment (meissadia) [inline src/common/Header.jsx:111]: This feels a bit dense with all the ternary operators.  Consider breaking things out into a few sub-components.

File: src/common/Header.jsx [hunk 5]
```diff
@@ -15,49 +25,99 @@ export const hideHeaderFooter = (path) => {
...
-          <nav className="nav">
-            <ul className="nav-primary">
+          <nav className="usa-nav">
+            <button type="button" className="usa-nav__close">
+              <img src={closeBtn} role="img" alt="Close" />
+            </button>
+            <ul className="usa-nav__primary usa-accordion">
               {links.map(link => {
                 let isActive = pathname.match(new RegExp('^' + link.href))
                 if(link.href === '/') isActive = link.href === pathname
```
Comment (meissadia) [inline src/common/Header.jsx:62]: This doesn't seem to work for the `Filing` page

**Review by meissadia — CHANGES_REQUESTED** [review]

File: src/common/Header.jsx (hunk 4 above)
Comment (Michaeldremy) [reply src/common/Header.jsx]: Going to leave this as is for now. Can come back and refactor later.

**Pull request merged**